cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -422,8 +422,8 @@ def dump(self, camel_case: bool = False) -> dict[str, Any]:
         return output
 
     @classmethod
-    def load(cls, data: dict | str) -> NodeApply:
-        data = json.loads(data) if isinstance(data, str) else data
+    def load(cls, resource: dict[Any, Any] | str, cognite_client: CogniteClient | None = None) -> NodeApply:
+        data = json.loads(resource) if isinstance(resource, str) else resource
```

Comment:
Note to self: merge error caused this, move changes into previous (squashed) commit.

Note 2: fix any-any